### ISSUE TITLE
Option type popup, trigger close, how to know it's save or close only?

### ISSUE THREAD

**Issue by qutub-bharmal** [ThemeFuse/Unyson#3686]
My options are as below:

`array(
   'myoptions' => array(
      'type' => 'popup',
      'label' => esc_html__( 'My Options' ),
      'button' => esc_html__( 'Generate' ),
      'popup-options' => array( // Options ),
      'custom-events' => array( 'close' => custom-event' )
   )
);`

Js trigger as below:
`fwEvents.debug(true)
fwEvents.on('fw:option-type:popup:custom:custom-event', function (data) {
   console.log(data);
});`

it works when I click on close button and also on save button, but in date, I didn't found anything which differentiates that this is save event or this is close event fires.

Basically, I want to perform some actin it saves button is clicked via ajax, so how can I identify this, can anyone please help.

**Comment by qutub-bharmal** [comment]
@ViorelEremia ???
